vllm/entrypoints/openai/api_server.py

```diff
@@ -692,8 +693,8 @@ async def reset_prefix_cache(raw_request: Request):
     @router.post("/sleep")
     async def sleep(raw_request: Request):
         # get POST params
-        level = raw_request.query_params.get("level", "1")
```

Comment:
i think the common practice is to use `query_params`? why do we need to change to `json` here?